app/views/welcome/events.html.slim

```diff
@@ -15,9 +15,18 @@
         | )
 
     .col.s12
-      ul.collection
+      ul.collection.events-list
+        - current_day = ''
         - cache('all_events') do
-          == render partial: 'shared/event_with_map', collection: @events, as: :event
+          - @events.each do |event|
```

Comment:
May be better to do a partial within another partial instead of using `.each`. This might end up using up a lot of memory when a lot of people hit the site. I guess we can use this first. I can clean it up later.